### PR TITLE
Add support for Speed Brain

### PR DESCRIPTION
**Pull request by cf-ypark**
## Description
Add support for Speed Brain feature

## Has your change been tested?
Unit tests have been added that align with the documented API spec.

## Types of changes

What sort of change does your code introduce/modify?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [x] This change is using publicly documented in [cloudflare/api-schemas](https://github.com/cloudflare/api-schemas) 
      and relies on stable APIs.

[1]: https://help.github.com/articles/closing-issues-using-keywords/

### CODE REVIEW

File: speed_brain.go
Comment (jeehoon-cf) [inline speed_brain.go:1]: Is this OK to be placed in root, not under /internal ?

Comment (cf-ypark) [reply speed_brain.go:1]: is there a reason why it should fall under /internal?

Comment (jeehoon-cf) [reply speed_brain.go:1]: I was looking at different repo.

**Review by jeehoon-cf — APPROVED** [review]

**Comment by cf-ypark** [comment]
not sure why `changelog-check` is failing @jacobbednarz any help?

**Comment by jacobbednarz** [comment]
i don't think we need this PR as it would already be supported using https://github.com/cloudflare/cloudflare-go/blob/master/zone.go#L928-L985. terraform we _should_ just be able to add the key to the array have this Just Work™

**Comment by cf-ypark** [comment]
got it, @jacobbednarz I'll close this one then. thanks!

**Pull request closed without merging**